specutils/io/default_loaders/twoslaq_lrg.py

```diff
@@ -0,0 +1,105 @@
+import astropy.io.fits as fits
+from astropy.units import Unit
+from astropy.wcs import WCS
+from specutils.io.registers import data_loader
+from specutils import Spectrum1D, SpectrumList
+
+
+def identify_2slaq_lrg(origin, *args, **kwargs):
+    """
+    Identify if the current file is a 2SLAQ-LRG file
+    """
+    with fits.open(args[0]) as hdulist:
```

Comment:
You may want to do the same here, to make sure the identifier function works with [file objects](https://github.com/astropy/specutils/blob/master/specutils/io/default_loaders/sdss.py#L33).

Reply:
Sure, done.